docs/markdown/spline_intro.md

```diff
@@ -93,7 +97,7 @@ Bézier curves have the following properties:
 
 .. math::
 
-  N_{1,p}(0)&=\underbrace{\frac{p!}{0!p!}}\_{=1}\underbrace{0^{0}}\_{=1}\underbrace{(1-0)^{p}}\_{=1}=1
+  N_{1,p}(0)&=\underbrace{\frac{p!}{0!p!}}_{=1}\underbrace{0^{0}}_{=1}\underbrace{(1-0)^{p}}_{=1}=1
```

Comment:
![Screenshot 2024-05-17 at 15 07 38](https://github.com/tataratat/splinepy/assets/47114801/b337c26d-af10-4406-bc74-acaa672fec86)
this one too

Reply:
And this:
<img width="1086" alt="Screenshot 2024-05-17 at 16 16 11" src="https://github.com/tataratat/splinepy/assets/81433605/654ee7dc-ea0c-4559-8cdb-72c1ef55829c">

Reply:
cannot reproduce error in local build @j042 on MacBook

Reply:
thanks for checking. Works fine after updating the dependencies